### PR TITLE
Tidy up static assets

### PR DESCRIPTION
**Pull request by ushkarev**
…by:
- removing support IE8-specific assets – this browser will not be supported (or tested with) so broken/partial support would be misleading
- linting frontend assets as well as server code
- normalising stylesheet sources to SCSS – it was already in use for an existing component and it can be linted by existing plugins unlike SASS
- making the url routes to static assets more structured:
    - only 1 route to each asset
    - mapping only built asset paths to urls (previously, build sources were served as well)
- removing externally-loaded jQuery UI library (because it’s unused and would be more appropriate packaged in-app)

### CODE REVIEW

File: assets/sass/local.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.govuk-main-wrapper {
+  min-height: 600px;
+}
```
Comment (xoen) [inline assets/sass/local.scss:3]: yay curly braces! 🎊

Comment (ushkarev) [reply assets/sass/local.scss:3]: but at least they're being linted now ;)

File: server/views/partials/layout.njk [hunk 2]
```diff
@@ -3,13 +3,6 @@
 
 {% block head %}
   <link href="/assets/stylesheets/application.css?{{ version }}" rel="stylesheet"/>
-
-  <script src="/assets/js/jquery.min.js"></script>
-  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
-          integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
-          crossorigin="anonymous"></script>
-  <link href="https://code.jquery.com/ui/1.12.1/themes/ui-lightness/jquery-ui.css" rel="stylesheet">
-
```
Comment (xoen) [inline server/views/partials/layout.njk:19]: This probably means that the CSP (Content Security Policy) could be simplified:

https://github.com/ministryofjustice/manage-my-prison/blob/main/server/middleware/setUpWebSecurity.ts

Comment (ushkarev) [reply server/views/partials/layout.njk:19]: ah thanks, i missed that

**Review by xoen — APPROVED** [review]

**Pull request merged**